nx.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "targetDefaults": {
+    "doc": {
+      "outputs": [
+        "{projectRoot}/data"
+      ],
+      "cache": true
+    }
+  },
+  "nxCloudId": "683f3ee2e6cc301ac6e3b787"
```

Comment:
i think we can scrap this as we don't want to use the nxCloud features for this. Let me know if you have a different opinion.